docs/adr/adr_statviz_fe_sharing.md

```diff
@@ -0,0 +1,148 @@
+
+We are giving here an overview over possible options for the user to share these visualizations - csv, svg, jpeg, iframe - and how much development and maintenance effort is needed in order to decide which options we should offer we should offer.
+
+Currently, we are planning to build these visualizations in a React FE with Nivo and Visx (libraries based on D3) and query the underlying data from a GraphQL endpoint.
+
+## Decision Drivers
+
+1. **Keep it simple**: Let's try to not add complexity to our codebase, especially for low maintenance sake.
+
+2. **Minimal development effort**: Keeping it simple might sometimes involve more effort to develop something.
+
+3. Create an effective foundation / re-usability for the planned future availability / aid ordering project
+
+4. Stay within DSEE 100x budget and avoid drawing from too many other resources (e.g. earmarked for other projects in Boxtribute) as much as possible
+
+## Considered options to share data
+
+### A. CSV
+
+#### What (minimal) work needs to be done on the FE side?
+
+1. We add a (fancy graphs) view in the Boxtribute v2 React App from where all visualizations can be downloaded.
+2. There should be options for filtering and grouping of the data in this view either globally or per visualization.
+3. We need to create a component which triggers the query and download.
+4. We need to create a function transforming the data from the graphQL query into csv format.
+
+#### What (minimal) work needs to be done on the BE side?
+
+1. We need to make the queries accessible through the private Boxtribute graphQL endpoint.
+
+#### comments
+
+Atm the filtering and grouping is happening on the FE side. The BE just returns all available data in the smallest granularity. There is the posibility to move filtering and grouping to the BE by making filters and grouping part of the graphQL query. This might make sense if we only allow the csv export.
+
+### B. SVG
+
+#### What (minimal) work needs to be done on the FE side?
+
+1. Steps 1, 2, 3 from CSV.
+2. There should be an input at least for the aspect ratio (or width and height) of the exported svg. This input could also just be given the window size of the current view. All other properties like axis label font size, axis thickness,... could be calculated based on this input. However, I can imagine that they might want more styling options that they can directly control. At the same time they could easily change the style in the exported svg through Adobe or any coding IDE.
+3. There needs to be a function transforming the data from the graphQL query into the svg. Since an svg can be easily included in the DOM, it makes sense to combine this with a preview view that is directly shown in the fancy graphs view even before the download.
```

Comment:
Both Nivo and VisX use SVG by default for their graphs.

So technically we are doing it the other way around, letting NIVO or VisX generating an SVG inside of the DOM and then taking it out of the dom to create a downloadable SVG